composer.json

```diff
@@ -50,10 +57,7 @@
         ]
     },
     "config": {
-        "bin-dir": "bin",
-        "platform": {
-            "php": "5.3.9"
```

Comment:
Well, this should be changed to ``5.5.9`` for the platform once the demo runs on Symfony 3.